### DISCUSSION TITLE
Expandable sidebar

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#2158] [Ideas]
Hello! It would be nice if it was possible to expand or if it had a side scroll on the left bar. This affects my experience when I have a lot of components.

![Sem título](https://user-images.githubusercontent.com/33108596/127359399-943542c3-23e9-431e-b858-5669a9a1b43a.jpg)

**Comment by joebudi** [comment]
@GuilhermeHirsch This is a design issue we need to solve as usage increases. We have a few ideas which we will tackle asap. Thanks for raising it and I will convert it to an issue!
